migration.sql

```diff
@@ -58,6 +58,8 @@ CREATE TABLE banking_stage_results_2.accounts_map_transaction(
   PRIMARY KEY (acc_id, transaction_id)
 );
 
+ALTER TABLE banking_stage_results_2.accounts_map_transaction ADD COLUMN is_atl BOOL;
```

Comment:
IMO we should start to append patches at the bottom at the file to make them applicable in sorted order